config/canonical.json

```diff
@@ -8915,14 +8917,19 @@
     "tags": {
       "amenity": "fuel",
       "brand": "Auchan",
+      "brand:wikidata": "Q758603",
+      "brand:wikipedia": "en:Auchan",
       "name": "Auchan"
     }
   },
   "amenity/fuel|Avanti": {
     "count": 106,
+    "countryCodes": ["at"],
     "tags": {
       "amenity": "fuel",
       "brand": "Avanti",
+      "brand:wikidata": "Q168238",
+      "brand:wikipedia": "en:OMV",
```

Comment:
Thanks for saying something. I was actually planning on correcting it in the morning as im pretty sure its AvantiGas and not OMV. I think I miss pasted it. Which us why I didnt just merge it. You asking about it is a good reminder though.

Reply:
I did find [Vertriebscenter Avanti](https://en.wikipedia.org/wiki/Vertriebscenter_Avanti) which redirects to the OMV page. Described as:

"Austrian chain of gas stations, operated by OMV Refining & Marketing GmbH"

Reply:
Oh. That explains it. I knew there was a connection to them somehow. I looked up the locations of the tags being used and its in Austria. So I guess its correct. The Wikipedia page could be vague about the connection, but whatever. A lot of the gas company entries seem to be seem that way. So, I'll just leave it as is. It might be worth revisiting latter though to see if theres any tagging overlap with the UK company.